lib/workers/repository/updates/generate.ts

```diff
@@ -115,7 +115,7 @@ export function generateBranchConfig(
         upgrade.newDigestShort ||
         upgrade.newDigest.replace('sha256:', '').substring(0, 7);
     }
-    if (upgrade.isDigest || upgrade.updateType === 'pin') {
+    if (upgrade.isDigest || upgrade.updateType === 'pinDigest') {
       upgrade.displayFrom = upgrade.currentDigestShort;
```

Comment:
This won't work, because if we are pinning digests then currentDigestShort is empty/null

Reply:
Yes I thought we want the change to be in the update table to be from nothing to the new digest.
Or are we expecting a different final result for this new updateType case ?

Reply:
I guess you could be right. It depends on how it will be displayed. I think something like `'' -> 'afe52ab8'` could look ok

Reply:
Yes that's how it looks.